server/data/resources.json

```diff
@@ -0,0 +1,32 @@
UN66aYp0YgICAgn+kysmbbTa2Qfhd7Rq9y89XNMeXp8nwvHb5J0z1DlJTy2Bd2Z4P1D8WxTVyVlzsvAzY/TcfRlwVIpvqAgICAgINazh486goZZWG0jrRRcpH/AEvVaHO9VSYqddohradQ5vJvrJJJ1kk3JJ2kk7SumgEYEBAQEBB9a4jWFmJ0xasWjUr6nm0uqsUv1OVyOP8AhzuPD1Uiqpftb1/IrW3omp8to+ipbIhGBBRsd1HvH/q1/Um84/2n7q1shbzm7y0NI5tPO79ncbNJ/cuJ2+gd43beKo8rjdcdVfP3dDh8ronot4+yZwVyXafUBBS94aCSQANpJsB4ozETPaFvFiMLzotlY48A9pPsuteqPdvbDkrG5rP9LpbI1rimHxVMUkMrQ+N40XA8OIO4g2IO4hZraazuCY25tyuydkw6pfA+5b50T7fORE6ncNIbCOI4ELp48kXrtXtXTDLdqICAgl9eZe5EF7h+KzQeY/V9U62+zd4Lat7V8K+bi4svzR/LbMKymils1/yb9ms90nk7d4+9WKZYs4/I+H5MfeveGdUqgICAgINDz0UDpcO023PYysmcB9Szo3HoO0v0BVjjW1dHk8IGV9CICAgICAgIKo36JBWYnUtclIvWYlklccKeymTd1HxWtkmLzP7SqWyMRgQUS7L8Dfw3+5a2902HvPT79la2RSIJezUZTmZnkcp78YvET9KIWGh1bq8LcCuTzMHTPXHiXY4PI6q9E+Y+yQ1RdEQR7lDibp5XC/caS1o3ajbSPElU8luqXo+Fx64qROu8sWtFxuuSOJula6N5u5lrE7S08eJFveFZw33GnB+IceMdotXxLYVM5zVs4uTAxGlc1oHbR3khPF1tcZPBw1dbHcpcOTot9Gto3DnO3EW5EWIPAjcV0kAjAgIJfXmXuRAQEGawXKF8Fmvu+P8Amb6J3jkpceSa9pUOVwK5fzU7T92701Q2RoewgtOsEKzExPhwb0tSem0al6rLUQEGr5x8aFLRSjUXzAwMBFx3mnTcRwDb+NuKscbF+JkiFblZvwse/VzvPDonlu/RdK9NSgwZoyV+ryWiYQEBAQEBAAvqWYjZM6jcsoArkOBM7lTLs8R/UFrbwkw/N/E/ZUtkYjAgIzE6namPZbhq/T3WWtZ9EuaO/V7qlshXWGV76eWOaM2exwcPDaDyIuDyK0yUi9ZrKTHeaWi0Oi8IxBlTDFOzzXtDhy4tPMG4PRcC9Jpaay9HjvF6xaPVeLVujPFaUxSyMO5xI5tJu0+xUrxqXqeNljJiraFqtUza8iKU/KSnYbMbzsbuPw96sYInW3G+KZIma09Y8trU7kiDn/O3gQpa50jRaOoBmHASXtKPaQ7110OPfqpr2QXjUtKU7QQEEvrzL3IgICAgyeBYu6mfruYye83/AJDn8VvS81VOXxYzV7fMkGKQOAcDcEXBG8HYVcidvOTExOpVIwIIRzq4v5RWmMHuQDsxw7Q2Mh/pb6i7HCx9OPfu4fPy9WTp9mmEA6jsVuY32VK3ms7hYz05brGsfBV749Oph5MZO09peKjWhGBAQEAIeO68p6fR1nbu5Kxjx67y5vJ5PV+WvhcKVRUybPZ8Qtb+EuL5v7+ypbIxGBAQU7D1FvEax+fsWviU0fmpP0+ypbIRBKeZvGbiakcfN+Wj9EkCRo5X0T6xXL5+PUxeHX+HZdxNJScue6ayxHC4qgAPbcjYRqcOhWtqxbymw8jJhndJY2HJKBpuS9w4EgD3AFRxhrC3b4nmmNRqGcijDQGtAAGoAagApnPmZmdyrRgQR7ntw7tKFs2+GVpv9iQ9mR+J0Z8FY41tX17tMnhBivoBAQS+vMvciAgICAg2vIzEfOgceLmdPpN/P2qxht6ON8Twa1lj+W1qdyXhX1TYYpZXeaxjpD0a0k/BZrHVMQ1tbprMuaaid0j3vcbue5z3Hi5xuT7SV6KI1GoeZtbqmZeay1EZeMlO08un6KO2OJWcfLvXtPeHg6kdusfco5xT6LdebSfPZQad3D4LX8O3sl/ycXueTuva3wTosf5GPW9vVlGd59i3jD7q9ubH6YXEcQbsClisQp5M98nmVa2QiCmTZ4j+oLW3hLi+b+/sqWyIQEBBTJs6a/YtbeEuKY6tT69lS28tJjU6EasxkjifktZTzXs0PDX/AHb+6+/QEnwUOenXjmE/Gv0ZIl0SuC9GICAgICDAZf0/aYbXtte0Ejx6TG6bfe0KTF88NbeHNK6auICCX15l7kQEBAQEFxh9T2Uscn1XAn0fpD2XWazqdos+P8THav0SaFeeVa3nIqezw6qI2uDY/B8jWn3Eqfi16ssQr8u3ThsgNd154RgQEBATbMd1LNfe47Om5a190mSdarHp91S2RCAgIKX7uo+N/wAlrZJj9Z+kqls0EYEBARlTFstwuP091livjSTL82/fuqWUQjLorJKv8oo6WUm5dG0OP22jRf8AzNK8/mp0ZJq9Jgv144ll1GlEBAQEFhlA29LVDjDKP/m5bV+aGJ8OWG7AutKs+rAIJfXmXuRAQEBAQfCjMJQw914ojxY0+1oV6vh5LLGr2j6y1PO07/D3c5Ix7yfyVzhf7YUOd/pn+EIrsuCICAgIKZNw4n3b/wC+a1t7JcXbdvZUtkYjAgICD47a3xPu/wCwtZ8wlr2rMvq2RCAgICD4Bt5/pZYiG823EfR9WWgjKas0NTp0BZ/DlezwIa/4vK4/OrrLt3OBbeLXs3dU10QEBAQYrKybs6Ktf9Wnnd7InFbUjdohifDl4Lqq76jAgl9eZe5EBAQEBARlJuG/Mw/ds/pCvV8Q8ll+e37z92r52B/h8n3kX9at8L/dCjzv9MoPXacAQEBAQLb/AO9yx6t4n8siy0EBAQEFO/w/Na+qT/p/yqWyMQEBAQEBAQS3mWPyFUP81v8Athcrn/PDs/Dvkn90jKg6IgICAg13OIf8Mr/uXjwOoqTF88fu1t4lzYumgEYEH//Z",
```

Comment:
I think this might not work as an image source, in the google search just from the right menu open the image into a new tap and you will get the actual URL.
Google uses this for search functions.

Reply:
thank you Ahmad for pointing it out. I have updated the file